src/DependencyInjection/AceEditorExtension.php

```diff
@@ -38,8 +41,47 @@ private function registerAceEditorParameters(array $config, ContainerBuilder $co
 
         $mode = 'src'.($debug ? '' : '-min').($config['noconflict'] ? '-noconflict' : '');
 
+        $useStimulus = false;
+        $bundles    = $container->getParameter('kernel.bundles');
+        assert(is_array($bundles));
+        if (in_array(StimulusBundle::class, $bundles, true) && interface_exists(AssetMapperInterface::class)) {
+            $useStimulus = true;
+        }
+
```

Comment:
There should be an if statement here to check if use_stimulus wasn't already set by the user in the config file. The logic that decides to use stimulus based on registered bundles should be a fallback logic. 
The reason for that is to let users explicitly decide if they would like or not to use stimulus even when bundles are installed.